### PR TITLE
Convert needlessly public methods and properties to internal.

### PR DESCRIPTION
**Pull request by dbart01**
The GraphQL support file is generally included in the module that also contains the generated files. There's no need to declare many `internal` methods and properties as `public`. Doing so increases the risk of end-users incorrectly using the API and pollutes the auto-complete namespace.

### CODE REVIEW

**Comment by dylanahsmith** [comment]
The tests fail since it is testing the support package as a separate package.

> Doing so increases the risk of end-users incorrectly using the API and pollutes the auto-complete namespace.

Is this something that has come up, or just an abstract problem?

If we want to distribute it as a swift package in the future, then we will need to revert these changes.

File: support/Sources/GraphQL.swift [hunk 1]
```diff
@@ -217,8 +221,8 @@ public class GraphQL {
 }
 
 public struct GraphQLResponse<DataType: GraphQL.AbstractResponse> {
-	public let data: DataType?
-	public let errors: [GraphQL.ResponseError]?
+	internal let data: DataType?
+	internal let errors: [GraphQL.ResponseError]?
```
Comment (raulriera) [inline support/Sources/GraphQL.swift]: These are important

File: codegen/lib/graphql_swift_gen/templates/type.swift.erb [hunk 2]
```diff
@@ -1,24 +1,48 @@
-// Generated from <%= script_name %>
+//
+//  <%= type.name %>.swift
+//  Buy
```
Comment (BenEmdon) [inline codegen/lib/graphql_swift_gen/templates/type.swift.erb]: Why are these instances of "Buy" sitting around?

File: codegen/lib/graphql_swift_gen/templates/ApiSchema.swift.erb [hunk 3]
```diff
@@ -1,4 +1,28 @@
...
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
```
Comment (BenEmdon) [inline codegen/lib/graphql_swift_gen/templates/ApiSchema.swift.erb]: Is the inline license necessary? Why not just license the whole repo?

Comment (dylanahsmith) [reply codegen/lib/graphql_swift_gen/templates/ApiSchema.swift.erb]: This is a license for the generated code, which will end up being in another repo.  I'm not sure if the license in the generated code is necessary though.

**Comment by BenEmdon** [comment]
Is the purpose of this PR to allow sending explicit nulls? This is what #8 and #12 are trying to achieve

**Comment by dbart01** [comment]
The purpose was originally to sync changes that we needed to make on Buy SDK. This was back in March. As for the recent commit with explicit null support, @sav007 and I had a long discussion about the kind of API we want to expose in Mobile Buy SDK and this is the outcome. You're more than welcome to adopt it as well.

**Pull request closed without merging**